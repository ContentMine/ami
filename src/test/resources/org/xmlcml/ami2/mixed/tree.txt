mixed
├── file0
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   └── scholarly.html
├── file1
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   └── scholarly.html
├── file2
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   └── scholarly.html
├── file3
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   └── scholarly.html
├── file4
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   └── scholarly.html
├── file5
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   └── scholarly.html
└── file6
    ├── fulltext.html
    ├── fulltext.pdf
    ├── fulltext.xml
    ├── results.json
    └── scholarly.html

