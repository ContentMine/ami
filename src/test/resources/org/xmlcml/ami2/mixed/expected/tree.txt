.
├── file0
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── results.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── empty.xml
│   │   │   └── genus
│   │   │       └── results.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── file1
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── empty.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── results.xml
│   │   │   └── genus
│   │   │       └── empty.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── file2
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── empty.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── results.xml
│   │   │   └── genus
│   │   │       └── empty.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── file3
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── empty.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── results.xml
│   │   │   └── genus
│   │   │       └── results.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── file4
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── results.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── results.xml
│   │   │   └── genus
│   │   │       └── results.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── file5
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── empty.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── results.xml
│   │   │   └── genus
│   │   │       └── results.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── file6
│   ├── fulltext.html
│   ├── fulltext.pdf
│   ├── fulltext.xml
│   ├── geneFiles.xml
│   ├── geneSnippets.xml
│   ├── genegeneSnippets.xml
│   ├── results
│   │   ├── gene
│   │   │   └── human
│   │   │       └── empty.xml
│   │   ├── sequence
│   │   │   └── dnaprimer
│   │   │       └── empty.xml
│   │   ├── species
│   │   │   ├── binomial
│   │   │   │   └── results.xml
│   │   │   └── genus
│   │   │       └── results.xml
│   │   └── word
│   │       └── frequencies
│   │           ├── results.html
│   │           └── results.xml
│   ├── results.json
│   ├── scholarly.html
│   ├── sequenceFiles.xml
│   ├── sequenceSnippets.xml
│   ├── speciesFiles.xml
│   ├── speciesSnippets.xml
│   ├── wordFiles.xml
│   └── wordSnippets.xml
├── geneFiles.xml
├── geneSnippets.xml
├── genegeneSnippets.xml
├── sequenceFiles.xml
├── sequenceSnippets.xml
├── speciesFiles.xml
├── speciesSnippets.xml
├── wordFiles.xml
└── wordSnippets.xml
